packages/ts-migrate-server/src/migrate/index.ts

```diff
@@ -60,7 +60,7 @@ export default async function migrate({
 
     const sourceFiles = project
       .getSourceFiles()
-      .filter(({ fileName }) => !/(\.d\.ts|\.json)$/.test(fileName));
+      .filter(({ fileName }) => !/(\.d\.ts|\.json)$|node_modules/.test(fileName));
```

Comment:
Perhaps `project.getSourceFiles()` shouldn't return files in `node_modules`. I looked at the docs `project.getSourceFiles()`, but they weren't clear to me on whether returning `node_modules` is intentional. If it's not, then perhaps the real fix is upstream.